packages/package_image_1_4_1/tool_dependencies.xml

```diff
@@ -0,0 +1,17 @@
+<tool_dependency>
+    <package name="Image" version="1.4.1">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://codeload.github.com/francescortiz/image/tar.gz/master</action>
+                <action type="make_directory">$INSTALL_DIR/lib/python</action>
```

Comment:
You should try to have an exact version of the code.
https://codeload.github.com/francescortiz/image/tar.gz/master is not guaranteed to contain version 1.4.1 of Image,
right?